commands/addons/attach.js

```diff
@@ -43,7 +43,7 @@ function * run (context, heroku) {
 module.exports = {
   topic: 'addons',
   command: 'attach',
-  description: 'attach add-on resource to a new app',
+  description: 'attach an existing add-on resource to an app',
```

Comment:
Maybe. Seems like a broader change though (as I think we say resource a lot in various places). I'd be open to discussing, but seems out of scope for this probably?